manifests/templates/release/olm/bundle/cdioperator.VERSION.csv.yaml.in

```diff
@@ -0,0 +1,116 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: cdioperator.{{.CsvVersion}}
+  namespace: placeholder
+  annotations:
+    capabilities: "Full Lifecycle"
+    categories: "Storage,Virtualization"
+    alm-examples: |
+      [
+        {
+          "apiVersion":"cdi.kubevirt.io/v1alpha1",
+          "kind":"CDI",
+          "metadata": {
+            "name":"cdi",
+            "namespace":"cdi"
+          },
+          "spec": {
+            "imagePullPolicy":"IfNotPresent"
+          }
+        }
+      ]
+    description: Creates and maintains CDI deployments
+spec:
+  displayName: CDI
+  description: |
+    CDI is a kubernetes extension that provides the ability to populate PVCs with VM images upon creation. Multiple image formats and sources are supported
+
+    _The CDI Operator does not support updates yet._
+  keywords:
+    - CDI
+    - Virtualization
+    - Storage
+  version: {{.CsvVersion}}
+  maturity: alpha
+{{.ReplacesCsvVersion}}
+  maintainers:
+  - name: KubeVirt project
+    email: kubevirt-dev@googlegroups.com
+  provider:
+    name: KubeVirt/CDI project
+  links:
+    - name: CDI
+      url: https://github.com/kubevirt/containerized-data-importer/blob/master/README.md
+    - name: Source Code
+      url: https://github.com/kubevirt/containerized-data-importer
+  icon:
+  - base64data: {{.CDILogo}}
+    mediatype: image/png
+  labels:
+    alm-owner-cdi: cdi-operator
+    operated-by: cdi-operator
+  selector:
+    matchLabels:
+      alm-owner-cdi: cdi-operator
+      operated-by: cdi-operator
+  installModes:
+    - type: OwnNamespace
+      supported: true
+    - type: SingleNamespace
+      supported: true
+    - type: MultiNamespace
+      supported: true
+    - type: AllNamespaces
+      supported: true
+  install:
+    strategy: deployment
+    spec:
+      clusterPermissions:
+        - serviceAccountName: cdi-operator
+          rules:
+{{.OperatorRules}}
+      deployments:
+        - name: cdi-operator
+          spec:
+{{.OperatorDeploymentSpec}}
+  customresourcedefinitions:
+    owned:
+      - name: cdis.cdi.kubevirt.io
+        version: v1alpha1
+        kind: CDI
+        displayName: CDI deployment
+        description: Represents a CDI deployment.
+        resources:
+          - kind: ConfigMap
+            name: cdi-operator-leader-election-helper
+            version: v1
+        specDescriptors:
+          - description: The ImagePullPolicy to use for the CDI components.
```

Comment:
More properties were added to the Spec since this PR was posted.  Docker repo/tak can now be specified.

Reply:
updated csv